crates/matrix-sdk-base/src/client.rs

```diff
@@ -143,7 +143,7 @@ impl BaseClient {
     ///   previous login call.
     pub fn with_store_config(config: StoreConfig) -> Self {
         let (room_info_notable_update_sender, _room_info_notable_update_receiver) =
-            broadcast::channel(100);
+            broadcast::channel((u16::MAX - 1) as usize);
```

Comment:
> This patch extends the capacity to 2^16 - 1.

That's 2¹⁶ - 2 though: 

> The largest value that can be represented by this integer type (2¹⁶ − 1). [^1]

[^1]: https://doc.rust-lang.org/std/primitive.u16.html#associatedconstant.MAX